geps/gep-1897/index.md

```diff
@@ -69,6 +80,9 @@ These are worthy goals that are already solved and thus will not be modified by
 2. HTTPS passthrough use cases (#2 in [Gateway API TLS Use Cases](#references))
 3. Termination of TLS for non-HTTP TCP streams (#3 in [Gateway API TLS Use Cases](#references))
 
+These existing use cases all relate to how a Gateway implementation should handle a downstream connection.
+This GEP, instead, focuses on the handling of upstream connections.
```

Comment:
Please use the agreed-upon terminology from https://gateway-api.sigs.k8s.io/geps/gep-2907/#frontend-and-backend.

```suggestion
These existing use cases relate to how a Gateway implementation should handle a frontend or passthrough connection.
This GEP, instead, focuses on the handling of backend connections.
```